tests/_data/manifests/src/integrator-manifests-master/Spryker/TestIntegratorAddConfigArrayElement/1.0.0/installer-manifest.json

```diff
@@ -1,37 +1,37 @@
 {
     "add-config-array-element": [
         {
-            "target": "\\Spryker\\Client\\TestIntegratorAddConfigArrayElement\\TestIntegratorAddConfigArrayElementConfig::getTestConfiguration",
+            "target": "\\Pyz\\Client\\TestIntegratorAddConfigArrayElement\\TestIntegratorAddConfigArrayElementConfig::getTestConfiguration",
```

Comment:
nope. We should have core namespace in the manifest. Project namespace is configurable by the project